Makefile

```diff
@@ -59,20 +64,23 @@ else
 endif
 
 # Default image repo
-QUAY_REGISTRY ?= quay.io/opencloudio
+QUAY_REGISTRY ?= quay.io/bitscuit
```

Comment:
Should we put images to your registry?

Reply:
Could we create another registry in opencloudio for testing?

Reply:
So the reason why we even need to use quay.io in the first place is because operator-sdk run bundle does not support private registries yet, otherwise I would have just used Artifactory. Yea, we can change this to use another registry in opencloudio. I was just using my own to test.

Reply:
new commit resolves this